docker.md

```diff
@@ -6,7 +6,7 @@
 
 1. Define the absolute parent path of repository bitcoinops.github.io .
     ```bash
-    repath=~/git/project/
+    repath=~/Desktop/
```

Comment:
I dont have a preference on this default location either way. 

Personally, I have a ~/Desktop/ folder on my mac, although terminal did ask for permission.

Reply:
> > > Had an additional thought here, can we add a heading below [Building The Site Locally](https://github.com/bitcoinops/bitcoinops.github.io#building-the-site-locally) where we briefly summarize docker build and link to this doc?
> > 
> > 
> > It should be a good way, and it should be obvious so people can know there is another way to build the site.
> 
> Feel free to add content and a link from the README to this new doc as part of this PR.

There are two ways
1. Separate "building" detail instruction from README.md to a file, and only provide two building links in README.md. or
2. Just like I updated in the README.md file. It is a simple way.